.buildkite/steps/packer.sh

```diff
@@ -17,14 +20,14 @@ echo "Packer image hash is $packer_hash"
 
 packer_file="${packer_hash}.packer"
 
-if ! aws s3 cp "s3://${BUILDKITE_AWS_STACK_BUCKET}/${packer_file}" . ; then
+if ! aws s3 cp "s3://${BUILDKITE_STACK_BUCKET}/${packer_file}" . ; then
   make build-ami | tee "$packer_file"
-  aws s3 cp "${packer_file}" "s3://${BUILDKITE_AWS_STACK_BUCKET}/${packer_file}"
+  aws s3 cp "${packer_file}" "s3://${BUILDKITE_STACK_BUCKET}/${packer_file}"
 else
   echo "Skipping packer build, no changes"
 fi
 
-image_id=$(grep -Eo "us-east-1: (ami-.+)$" "$packer_file" | awk '{print $2}')
-echo "AMI for us-east-1 is $image_id"
+image_id=$(grep -Eo "${AWS_DEFAULT_REGION}: (ami-.+)$" "$packer_file" | awk '{print $2}')
```

Comment:
Should this be `$AWS_REGION`?

Reply:
@lox - I've pushed a rebase now.

`${AWS_DEFAULT_REGION}` vs `${AWS_REGION}`. Both are available at this point, and `${AWS_REGION}` is probably more appropriate so I'll test and update.